mne/annotations.py

```diff
@@ -85,6 +85,14 @@ def __init__(self, onset, duration, description,
         self.duration = duration
         self.description = np.array(description, dtype=str)
 
+    def __repr__(self):
+        """Show the representation."""
+        kinds = sorted(set(['"%s"' % d.split(' ')[0]
+                            for d in self.description]))
+        kinds = ', '.join(kinds[:3]) + ('' if len(kinds) <= 3 else '...')
```

Comment:
good that annotations have repr now, what do you think about adding how many annotations are of each kind - for example in brackets? Not crucial but may be useful.